src/test/java/misc/migration/v1_0/EBasic.java

```diff
@@ -0,0 +1,116 @@
+package misc.migration.v1_0;
```

Comment:
These are the test models.
The whole test tries to migrate from 1.0 to 1.1. to 1.2.

1.0 & 1.2 are nearly the same, so that both directions (addd/drop) of all migration features are tested.